Taarafo.Core.Tests.Unit/Services/Foundations/GroupPosts/GroupPostServiceTests.Exception.Add.cs

```diff
@@ -0,0 +1,60 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient;
+using Moq;
+using Taarafo.Core.Models.GroupPosts.Exceptions;
+using Taarafo.Core.Models.GroupPosts;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.GroupPosts
+{
+    public partial class GroupPostServiceTests
```

Comment:
Name should be in plural:
Yours: `GroupPostServiceTests.Exception.Add.cs`
Desired: `GroupPostServiceTests.Exceptions.Add.cs`